src/components/client/ClientsDetail.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import ClientInfo from '../project/ClientInfo';
+import ProjectSummary from './ProjectSummary';
+import Spinner from '../global/Spinner';
+import {config} from '../../util/config.js';
+import './ClientDetailsCard.css';
+
+export default class ClientsDetail extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      clients: {},
+      id: props.id,
+      loading: true,
+    };
+    this.getClient = this.getClient.bind(this);
+  }
+
+  getClient() {
+    fetch(`${config.apiUrl}/clients/${this.state.id}`)
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          clients: data.client,
+          loading: false,
+        });
+      })
+      .catch(error => console.log(error));
+  }
+
+  componentDidMount() {
+    this.setState({id: this.props.id});
+    this.getClient();
+  }
+
+  render() {
+    if (this.state.clients === {}) {
+      return null;
+    }
```

Comment:
I cant see why we need line 37 to 39

Reply:
Neither was Jenna's code so maybe she needed it?

Reply:
🤣 I think you can remove it

Reply:
Okay will do

Reply:
Done!